HoneyBadgers.WebApp/Views/Shared/_Layout.cshtml

```diff
@@ -31,6 +31,9 @@
                         <li class="nav-item">
                             <a class="nav-link" asp-area="" asp-controller="Movie" asp-action="Index">Movies</a>
                         </li>
+                        <li class="nav-item">
+                            <a class="nav-link" asp-area="" asp-controller="FavoriteMovie" asp-action="Index">FavoriteMovies</a>
```

Comment:
to się wyświetla użytkownikowi więc polecałabym Favorite Movies ze spacją pomiędzy